2darraysearch.c

```diff
@@ -0,0 +1,57 @@
+#include<stdio.h>

```

Comment:
ok sir i put the title wrong, this was one of the exercise question given by @viveksinghgaur sir after 2nd week.